react-components/src/utils/hooks/useScrollIntoView.js

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ *
+ *    Copyright 2020 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+import { useEffect } from 'react';
+
+const OPTIONS_DEFAULTS = { behavior: 'smooth', block: 'center' };
+
+/**
```

Comment:
Shouldn't we add a note or copyright header here, since this is taken from PWA studio?

Reply:
The original component doesn't have a copyright header - https://github.com/magento/pwa-studio/blob/develop/packages/peregrine/lib/hooks/useScrollIntoView.js
I'll add a link to the original code

Reply:
I think that's a problem actually. IIUC Magento uses OSL 3.0 which requires to publish any used code under OSL 3.0 as well which is incompatible with our Apache 2.0 license. I think we need to double check that.

Reply:
Yes, they use OSL 3.0. 
I'll revert the change and use a different method.